crates/ruma-common/src/serde/strings.rs

```diff
@@ -89,16 +111,16 @@ where
 }
 
 #[derive(Deserialize)]
-struct F64OrStringWrapper(#[serde(deserialize_with = "deserialize_as_f64_or_string")] f64);
+struct NumberOrStringWrapper(#[serde(deserialize_with = "deserialize_as_number_or_string")] f64);
```

Comment:
Since this is a breaking change to `ruma-common`, can you add a changelog entry?

Reply:
Sure!